src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double side;
+
+    public RightTriangle(Color color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.pow(side, 2) * Math.sqrt(3) / 4;
```

Comment:
Consider defining `Math.sqrt(3) / 4` as a constant to avoid magic numbers and improve readability.